Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -155,180 +154,6 @@ class IEditingSchema(Interface):
         required=False)
 
 
```

Comment:
I would add here also a `zope.deferredimport.deprecated` in case someone imported from here.

Reply:
Yes, it seems that this is necessary, I had it first there and then thought, that the API is always imported from `Products.CMFPlone.interfaces`